lic/client/src/start.cpp

```diff
@@ -0,0 +1,31 @@
+#include <ros/ros.h>
+#include "client/name.h"
+#include "client/topic_name.h"
+
+void nameInfoCallback(const client::name::ConstPtr& msg) {
+    ROS_INFO("Subcriber Info: name:%s", msg->name.c_str());
+}
+
+int main(int argc, char **argv) {
+    ros::init(argc, argv, "start");
+    ros::NodeHandle n;
+    ros::service::waitForService("/show_topic");
+    ros::ServiceClient client_name = n.serviceClient<client::topic_name>("/show_topic");
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]